frontend/src/app/main/ui/onboarding/team_choice.cljs

```diff
@@ -54,27 +51,30 @@
      [:p {:class (stl/css :modal-desc)}
       (tr "onboarding.team-modal.create-team-feature-5")]]]])
 
-(def ^:private schema:invite-form
-  [:map {:title "InviteForm"}
-   [:role :keyword]
-   [:emails {:optional true} [::sm/set ::sm/email]]])
-
 (defn- get-available-roles
   []
   [{:value "viewer" :label (tr "labels.viewer")}
    {:value "editor" :label (tr "labels.editor")}
    {:value "admin" :label (tr "labels.admin")}])
 
-(mf/defc team-form-step-2
-  {::mf/props :obj}
-  [{:keys [name on-back go-to-team?]}]
+(def ^:private schema:team-form
+  [:map {:title "TeamForm"}
+   [:name [::sm/text {:max 250}]]
+   [:role :keyword]
+   [:emails {:optional true} [::sm/set ::sm/email]]])
+
+(mf/defc team-form
+  {::mf/props :obj
+   ::mf/private true}
+  [{:keys [go-to-team?]}]
```

Comment:
The same here for call conventions (also, no `?` is allowed in component props as convention)
I know that this is not new component, but this PR makes substantial changes to the component, so I think we also can improve it and migrate it to new style.